server/lib/nl/config_builder/base.py

```diff
@@ -208,11 +237,14 @@ def decorate_chart_title(title: str,
       else:
         title = title + ' in ' + place.name
 
+  # Use title case.
+  title = _replace_special(title.title())
+
   if add_date:
     title = title + ' (${date})'
 
   if title_suffix:
-    title += ' - ' + title_suffix
+    title += ' - ' + _replace_special(title_suffix.title())
 
   return title
```

Comment:
nit: maybe do this at the very end?

Reply:
In this case, the title casing needs to happen carefully because we don't want to capitalize ${date} :)

Reply:
Hmm, add that comment?

Reply:
Added the comment.